Deployment/terraform_conditional_run.ps1

```diff
@@ -53,6 +53,7 @@ $terraformOutput = terraform output -json | ConvertFrom-Json
 
 write-output "Set JSON output into pipeline variables"
 Write-Host "##vso[task.setvariable variable=MsiWafUrl;isOutput=true]$env:SERVICE_DNS_URL"
+Write-Host "##vso[task.setvariable variable=Admin_DNS_URL;isOutput=true]$env:SERVICE_ADMIN_DNS_URL"
```

Comment:
The value for SERVICE_ADMIN_DNS_URL is coming from a variable library (SERVICE_ADMIN_DNS_URL from Maritime-Safety-Information-XX-Variables) so I don't understand why this script needs to export the same value again (but called Admin_DNS_URL)